MailCore/Models/Contact.swift

```diff
@@ -68,17 +60,14 @@ public class Contact: Object, Codable, Identifiable {
             id = try values.decode(String.self, forKey: .id)
         }
         color = try values.decode(String.self, forKey: .color)
-        // Will need to investigate this later
```

Comment:
I removed this as we do not use this and I find it cleaner. Let me know if I should keep it